auth.go

```diff
@@ -559,7 +592,14 @@ func (v *Verifier) FullVerify(
 	request protocol.AuthorizationRequestMessage,
 	opts ...pubsignals.VerifyOpt, // TODO(illia-korotia): is ok have common option for VerifyJWZ and VerifyAuthResponse?
 ) (*protocol.AuthorizationResponseMessage, error) {
-
+	cfg := pubsignals.VerifyConfig{}
+	for _, o := range opts {
+		o(&cfg)
+	}
+	if (cfg.AllowExpiredMessages == nil || !*cfg.AllowExpiredMessages) &&
+		request.ExpiresTime != nil && time.Now().After(time.Unix(*request.ExpiresTime, 0)) {
+		return nil, errors.New("Authorization request message is expired")
+	}
```

Comment:
VerifyAuthResponse is called in this function, don't need to write check twice

Reply:
Should we verify response message inside VerifyAuthResponse ?
So in full verify it checks request and response verified inside VerifyAuthResponse.

Reply:
ah..ok maybe you are right